Error reading from serial port

Every time OH is shut down, I get the following error...

```
2019-08-21 08:46:11.132 [ERROR] [org.openhab.binding.zigbee.handler.ZigBeeSerialPort] - Error reading from serial port 
java.lang.InterruptedException: null
        at java.lang.Object.wait(Native Method) ~[?:?]
        at org.openhab.binding.zigbee.handler.ZigBeeSerialPort.read(ZigBeeSerialPort.java:267) [262:org.openhab.binding.zigbee:2.5.0.201908112253]
        at org.openhab.binding.zigbee.handler.ZigBeeSerialPort.read(ZigBeeSerialPort.java:243) [262:org.openhab.binding.zigbee:2.5.0.201908112253]
        at com.zsmartsystems.zigbee.dongle.ember.internal.ash.AshFrameHandler.getPacket(AshFrameHandler.java:281) [225:com.zsmartsystems.zigbee.dongle.ember:1.2.1]
        at com.zsmartsystems.zigbee.dongle.ember.internal.ash.AshFrameHandler.access$200(AshFrameHandler.java:56) [225:com.zsmartsystems.zigbee.dongle.ember:1.2.1]
        at com.zsmartsystems.zigbee.dongle.ember.internal.ash.AshFrameHandler$1.run(AshFrameHandler.java:164) [225:com.zsmartsystems.zigbee.dongle.ember:1.2.
```

| Configuration    | Description                                                                    |
|------------------|--------------------------------------------------------------------------------|
| Coordinator used | HUSBZB-1 |
| openHAB version  | S1658 (but this has been happening since for a few builds now, probably since 1.2 libraries were added  |

```
223 x Active x  80 x 1.2.1                 x com.zsmartsystems.zigbee
224 x Active x  80 x 1.2.1                 x com.zsmartsystems.zigbee.dongle.cc2531
225 x Active x  80 x 1.2.1                 x com.zsmartsystems.zigbee.dongle.ember
226 x Active x  80 x 1.2.1                 x com.zsmartsystems.zigbee.dongle.telegesis
227 x Active x  80 x 1.2.1                 x com.zsmartsystems.zigbee.dongle.xbee
262 x Active x  80 x 2.5.0.201908112253    x org.openhab.binding.zigbee
263 x Active x  80 x 2.5.0.201908112253    x org.openhab.binding.zigbee.cc2531
264 x Active x  80 x 2.5.0.201908112253    x org.openhab.binding.zigbee.ember
265 x Active x  80 x 2.5.0.201908112253    x org.openhab.binding.zigbee.telegesis
266 x Active x  80 x 2.5.0.201908112253    x org.openhab.binding.zigbee.xbee
```

--- a/org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeSerialPort.java
+++ b/org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeSerialPort.java
@@ -181,7 +181,8 @@ public class ZigBeeSerialPort implements ZigBeePort, SerialPortEventListener {
             } catch (RuntimeException e) {
                 if (!portOpenRuntimeExcepionMessages.contains(e.getMessage())) {
                     portOpenRuntimeExcepionMessages.add(e.getMessage());
-                    logger.error("Serial Error: Device cannot be opened on Port {}. Caused by {}", portName, e.getMessage());
+                    logger.error("Serial Error: Device cannot be opened on Port {}. Caused by {}", portName,
+                            e.getMessage());
                 }
                 return false;
             }
@@ -269,7 +270,6 @@ public class ZigBeeSerialPort implements ZigBeePort, SerialPortEventListener {
             }
             return -1;
         } catch (InterruptedException e) {
-            logger.error("Error reading from serial port ", e);
         }
         return -1;
     }

